pkg/products/cloudresources/reconciler.go

```diff
@@ -174,6 +179,50 @@ func (r *Reconciler) Reconcile(ctx context.Context, installation *integreatlyv1a
 	return integreatlyv1alpha1.PhaseCompleted, nil
 }
 
+func (r *Reconciler) removeSnapshots(ctx context.Context, installation *integreatlyv1alpha1.RHMI, client k8sclient.Client) (integreatlyv1alpha1.StatusPhase, error) {
+
+	logrus.Infof("Removing postgres and redis snapshots")
+
+	pgSnaps := &crov1alpha1.PostgresSnapshotList{}
+	listOpts := []k8sclient.ListOption{
+		k8sclient.InNamespace(installation.Namespace),
+	}
+	err := client.List(ctx, pgSnaps, listOpts...)
+	if err != nil {
+		logrus.Error("Failed to list postgres snapshots")
+		return integreatlyv1alpha1.PhaseFailed, fmt.Errorf("failed to list postgres snapshots: %w", err)
+	}
+
+	for _, pgSnap := range pgSnaps.Items {
+		logrus.Infof("bgbg Deleting %s", pgSnap.Name)
+		if err := client.Delete(ctx, &pgSnap); err != nil {
+			logrus.Infof("Failed to delete postgres snapshot %s", pgSnap.Name)
+			return integreatlyv1alpha1.PhaseFailed, err
+		}
+	}
+
+	redisSnaps := &crov1alpha1.RedisSnapshotList{}
+	listOpts = []k8sclient.ListOption{
+		k8sclient.InNamespace(installation.Namespace),
+	}
+	err = client.List(ctx, redisSnaps, listOpts...)
+	if err != nil {
+		logrus.Error("Failed to list redis snapshots")
+		return integreatlyv1alpha1.PhaseFailed, fmt.Errorf("failed to list redis snapshots: %w", err)
+	}
+
+	for _, redisSnap := range redisSnaps.Items {
+		if err := client.Delete(ctx, &redisSnap); err != nil {
```

Comment:
Should we include a log statement here similar to postgres above? `logrus.Infof("Deleting %s", pgSnap.Name)`